packages/react/tabs/src/Tabs.tsx

```diff
@@ -140,77 +135,60 @@ TabsList.displayName = TAB_LIST_NAME;
  * -----------------------------------------------------------------------------------------------*/
 
 const TAB_NAME = 'TabsTab';
+const TAB_DEFAULT_TAG = 'div';
 
 type TabsTabOwnProps = Polymorphic.Merge<
-  Polymorphic.OwnProps<typeof Primitive>,
+  Omit<Polymorphic.OwnProps<typeof RovingFocusItem>, 'focusable' | 'active'>,
   {
     value: string;
     disabled?: boolean;
   }
 >;
 
-type TabsTabPrimitive = Polymorphic.ForwardRefComponent<
-  Polymorphic.IntrinsicElement<typeof Primitive>,
-  TabsTabOwnProps
->;
+type TabsTabPrimitive = Polymorphic.ForwardRefComponent<typeof TAB_DEFAULT_TAG, TabsTabOwnProps>;
 
 const TabsTab = React.forwardRef((props, forwardedRef) => {
-  const { value, disabled, ...tabProps } = props;
+  const { as = TAB_DEFAULT_TAG, value, disabled = false, ...tabProps } = props;
   const context = useTabsContext(TAB_NAME);
   const tabId = makeTabId(context.baseId, value);
   const tabPanelId = makeTabsPanelId(context.baseId, value);
   const isSelected = value === context.value;
-  const rovingFocusProps = useRovingFocus({ disabled, active: isSelected });
   const handleTabChange = useCallbackRef(() => context.onValueChange(value));
 
-  const handleKeyDown = composeEventHandlers(
-    tabProps.onKeyDown,
-    composeEventHandlers(rovingFocusProps.onKeyDown, (event) => {
-      if (!disabled && (event.key === ' ' || event.key === 'Enter')) {
-        handleTabChange();
-      }
-    })
-  );
-
-  const handleMouseDown = composeEventHandlers(
-    tabProps.onMouseDown,
-    composeEventHandlers(rovingFocusProps.onMouseDown, (event) => {
-      // only call handler if it's the left button (mousedown gets triggered by all mouse buttons)
-      // but not when the control key is pressed (avoiding MacOS right click)
-      if (!disabled && event.button === 0 && event.ctrlKey === false) {
-        handleTabChange();
-      }
-    })
-  );
-
-  const handleFocus = composeEventHandlers(
-    tabProps.onFocus,
-    composeEventHandlers(rovingFocusProps.onFocus, () => {
-      // handle "automatic" activation if necessary
-      // ie. activate tab following focus
-      const isAutomaticActivation = context.activationMode !== 'manual';
-      if (!isSelected && !disabled && isAutomaticActivation) {
-        handleTabChange();
-      }
-    })
-  );
-
   return (
-    <Primitive
+    <RovingFocusItem
       role="tab"
       aria-selected={isSelected}
       aria-controls={tabPanelId}
       aria-disabled={disabled || undefined}
       data-state={isSelected ? 'active' : 'inactive'}
       data-disabled={disabled ? '' : undefined}
-      data-orientation={context.orientation}
       id={tabId}
+      focusable={!disabled}
+      active={isSelected}
```

Comment:
Do we want to move these below `...tabProps` like we did with `name` in `RadioGroup` because we are omitting them from the types?

Reply:
Yeah, can do. Do you think we'd need a similar comment?

Reply:
I don't think so personally, because i think that when things are moved below props spread, they are there for a reason. But I know you like to keep things consistently ordered so might be more of an issue for you looking at your own code later?